.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+Use the following template to create your pull request
+
+## Description
+
+Describe the changes that where made in this pull request. When possible start with a user story - short, simple descriptions of a feature told from the perspective of the person who desires the new capability. Be sure to also include the following information:
+
+1. Motivation for change
+2. What was changed
+3. How does this impact application developers
+4. Link to relevant issues and documentation
+5. Provide examples of use cases with code samples and applicable acceptance criteria
```

Comment:
Generally speaking, PRs that aren't part of the SIP process are going to be for (comparatively) very small changes.  Most non-trivial features should be implemented via the SIP process.

Reply:
Added a comment at the top of the PR template to address this concern.